group_vars/cinder_api_container.yml

```diff
@@ -13,8 +13,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-container_skel:
-  repo_container:
-    properties:
-      service_name: repo
-      lxc_container_variant: "repo_server-{{ rpc_release }}"
+lxc_container_variant: "cinder-{{ rpc_release }}"
```

Comment:
Any reason why you're not using the 'cinder_all' group here? The variant is the same for all cinder containers and this var will only apply to containers anyway.